lib/mongo/client_encryption.rb

```diff
@@ -19,6 +19,33 @@ module Mongo
   # and creating data keys.
   class ClientEncryption
 
+    # A class that implements I/O methods between the driver and
+    # the MongoDB server or mongocryptd.
+    #
+    # This class should have its own file, just leaving it here
+    # for simplicity.
+    #
+    # @api private
+    class IO
```

Comment:
I would like to reconsider this class name prior to release, since IO is also a standard library module.

Reply:
Yeah that's a great point. I will make a todo to change.